pom.xml

```diff
@@ -2755,12 +2755,13 @@
                                 <bannedDependencies>
                                     <excludes combine.children="append">
                                         <exclude>*:*_2.10</exclude>
+                                        <exclude>*:*_2.12</exclude>
```

Comment:
Does the change version script switch this over to _2.11?  (And hence do the right thing?)

Reply:
yes!

Reply:
https://github.com/locationtech/geomesa/pull/2682/files#diff-426d95ee0ba39aed84d2060724c557dff39626437bf23ef0dca67d7fdb7fa6f8R77